mirage/factories/institutional-user.ts

```diff
@@ -5,17 +5,24 @@ import InstitutionalUser from 'ember-osf-web/models/institutional-user';
 export default Factory.extend<InstitutionalUser>({
     institution: association() as InstitutionalUser['institution'],
     user: association() as InstitutionalUser['user'],
-    userFullName() {
-        return `${faker.name.firstName()} ${faker.name.lastName()}`;
-    },
     department() {
-        return faker.name.jobTitle();
+        return faker.random.arrayElement(['Architecture', 'Biology', 'Psychology']);
     },
     publicProjectCount() {
-        return faker.random.number();
+        return faker.random.number({ min: 0, max: 99 });
     },
     privateProjectCount() {
-        return faker.random.number();
+        return faker.random.number({ min: 0, max: 99 });
+    },
+    afterCreate(institutionalUser, server) {
+        if (!institutionalUser.userFullName && !institutionalUser.userGuid) {
```

Comment:
Just curious, did
```ts
if (!institutionalUser.user) {
```
not work?

Reply:
It looks like institutionalUser.user was always defined here, so the table was never being populated when running with mirage. I just changed this to check the actual properties I was using in the tests that I needed manual control of.